app/views/editor/_editor.html.erb

```diff
@@ -17,7 +17,7 @@
   </div>
 </div>
 
-<div class="well" id="preview-main" style="display:none;"> </div>
+<div class="well" id="comment-preview-main" style="display:none;"> </div>
 
```

Comment:
I know this is kind of weird on the wiki editor to have this ID be #comment-preview-main, but I have a master plan for this and will take care of it in the next few PRs.

Basically with an OOP refactor of `editor.js`, it will be possible to `$E.initialize({default main editor form ID})`.